llvm/lib/Target/Hexagon/HexagonISelLowering.h

```diff
@@ -362,6 +362,7 @@ class HexagonTargetLowering : public TargetLowering {
   shouldExpandAtomicRMWInIR(AtomicRMWInst *AI) const override {
     return AtomicExpansionKind::LLSC;
   }
+  bool softPromoteHalfType() const override { return true; }
```

Comment:
I suppose this is technically an ABI change, thus precluding a backport to 20.1.1, right?

Reply:
@tstellar is this kind of change one that prevents backporting?  Or requires a 20.2.x?